File: old.eFromAPI/Plugins/Customers.Pn/Customers.Pn/Infrastructure/Models/Customer/CustomerPnFullModel.cs
using System;

namespace Customers.Pn.Infrastructure.Models.Customer
{
    public class CustomerPnFullModel
    {
        public int Id { get; set; }
        public DateTime CreatedDate { get; set; }
        public string CreatedBy { get; set; }
        public int? CustomerNo { get; set; }
        public string CompanyName { get; set; }
        public string CompanyAddress { get; set; }
        public string ZipCode { get; set; }
        public string CityName { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public string ContactPerson { get; set; }
        public string Description { get; set; }
    }
}

File: eform-client/src/app/plugins/modules/customers-pn/models/customer/customer-pn-full.model.ts
export class CustomerPnFullModel {
  id: number;
  createdBy: string;
  companyName: string;
  companyAddress: string;
  companyAddress2: string;
  customerNo: string;
  zipCode: string;
  cityName: string;
  phone: string;
  email: string;
  contactPerson: string;
  description: string = null;
  relatedEntityId: string;
  eanCode: string;
  vatNumber: string;
  countryCode: string;
  crmId: number;
  cadastralNumber: string;
  propertyNumber: number;
  apartmentNumber: number;
  completionYear: number;
  floorsWithLivingSpace: number;
}
